### ISSUE TITLE
v0.6.2 Release

### ISSUE THREAD

**Issue by dbent** [BahamutoD/BDAnimationModules#1]
There's a tag for [v0.6.2](https://github.com/BahamutoD/BDAnimationModules/releases/tag/v0.6.2) but no actual release, meaning CKAN is stuck at v0.6.1. Is v0.6.1 the correct last version or not?

**Comment by BahamutoD** [comment]
No, the newest is 0.6.2 (packaged with Adjustable Landing Gear), but when I
uploaded it before, it wouldn't get marked as the latest version and there
was some other issue.  I haven't gotten around to fixing it,  but I will as
soon as I can.
